src/ui/popup/index.tsx

```diff
@@ -72,16 +72,16 @@ if (DEBUG) {
         try {
```

Comment:
Please revert these lines. Code-style should take of styling issue. Also not related to adding dark themes PR. Don't clutter 1 PR with multiple things.

Reply:
Okay, is there a label or something called "Code-style"? should I do these lines there?